### DISCUSSION TITLE
Error : Installing atri results in failed parsing requirement

### DISCUSSION THREAD

**Discussion by guptaishika128** [Atri-Labs/atrilabs-engine#584] [Help: Installation & Start] [answered]
![image](https://user-images.githubusercontent.com/78978568/208284785-a227848e-11db-4421-a68b-d0c8cc1d23ef.png)
![image](https://user-images.githubusercontent.com/78978568/208284802-d20d3fb1-6cd6-4ebc-b951-a6c1d966dccf.png)
It ends with Failed parsing requirement. Tried uninstalling and reinstalling pipenv but nothing worked.

**Comment by cruxcode** [comment]
@guptaishika128 please copy this command `pipenv install atri==0.0.78`

You have incorrectly used space around `==`.

**Reply by guptaishika128** [reply]
![image](https://user-images.githubusercontent.com/78978568/208286325-d184447c-050c-4cd5-8420-2ee2531e2e05.png)
![image](https://user-images.githubusercontent.com/78978568/208286332-d2ad5a0b-909b-46fe-9372-964aee63b2ad.png)
Now it's showing this error

**Reply by cruxcode** [reply]
It seems the prompt library we are using in python is expecting a windows console, but, since you are using Git Bash (`xterm` based shell), the prompt library is throwing an error. One simple workaround would be to use window's cmd instead of Git Bash.

**Comment by darshitac11** [comment, marked as answer]
Use CMD instead of Git Bash / Powershell in Windows.
